app/views/static_pages/splash_page.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="layouts">
+  <div class="parent">
+      <div class="card" id="leftToRight">
+          <h2>Money Manager</h2>
+          <div class="buttons">
```

Comment:
- [ ] You have made a great effort in sticking to the template design just like the previous reviewer suggested. However, I still need to point out that you're not yet using the correct font family for the `h2` element. Please make this change 👍 
## Design Guideline
![image](https://user-images.githubusercontent.com/46839548/197299377-3e189dfa-ab69-472c-a084-11663933f798.png)
## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/197299117-2f9af455-88af-4a87-85af-18a01b5ccd1e.png)

Reply:
I have already done done I updated the sass file to import the fertigo font and the h2 has a font family of it, maybe it's not working on your end